.github/workflows/update-parsers-pr.yml

```diff
@@ -35,6 +35,11 @@ jobs:
           cp lockfile.json /tmp/lockfile.json
           cat /tmp/lockfile.json | /tmp/jq --sort-keys > lockfile.json
 
+      - name: Commit changes
+        run: |
+          git add lockfile.json
+          git commit -m "Update lockfile.json"
+
```

Comment:
how does this change anything in the logic? Won't the create pull request action do another commit?

Reply:
Apparently not, given the example showed in the `README`

Reply:
:shrug: ok

Reply:
Lol yeah, let's try that, it wont hurt anyway :wink: